app/upgrades/v3/upgrade.go

```diff
@@ -19,12 +27,40 @@ func GetStoreUpgrades() *storetypes.StoreUpgrades {
 func CreateUpgradeHandler(
 	mm *module.Manager,
 	configurator module.Configurator,
+	rollappkeeper RollappKeeper,
+	seqkeeper SequencerKeeper,
+	dakeeper DelayedAckKeeper,
 ) upgradetypes.UpgradeHandler {
-	return func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+	return func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
 		logger := ctx.Logger().With("upgrade", UpgradeName)
 
+		// reduce the gauge creation fee and remove `add to gauge` fee
+		DYM := sdk.NewIntFromBigInt(new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil))
+		incentivestypes.CreateGaugeFee = DYM.Mul(sdk.NewInt(10))
+		incentivestypes.AddToGaugeFee = sdk.ZeroInt()
+
 		// Start running the module migrations
 		logger.Debug("running module migrations ...")
-		return mm.RunMigrations(ctx, configurator, vm)
+		newVM, err := mm.RunMigrations(ctx, configurator, fromVM)
+		if err != nil {
+			return newVM, err
+		}
```

Comment:
Possibly stylistic, but why not just just `return mm.RunMigrations(ctx, configurator, vm)` at the end of the function, rather than adding unnecessary error handling here?

Reply:
Or maybe wrap the err with a message